Source/dafny.msbuild/build/dafny.msbuild.targets

```diff
@@ -16,8 +16,8 @@
             <DafnyOutputFile>$(IntermediateOutputPath)GeneratedFromDafny.cs</DafnyOutputFile>
         </PropertyGroup>
         <Message Text="Compiling Dafny source files to $(DafnyOutputFile)..." Importance="high"/>
-        <Exec Command="$(DafnyBinariesDir)$(Dafny) /out:$(DafnyOutputFile) @(DafnySource, ' ') /compile:0 /spillTargetCode:3 /noVerify" />
-        <ItemGroup>
+		<Exec Command="$(DafnyBinariesDir)$(Dafny) /out:$(DafnyOutputFile) @(DafnySource, ' ') @(CompileDafnyPassthrough, ' ') /compile:0 /spillTargetCode:3 /noVerify" />
+		<ItemGroup>
```

Comment:
No tabs please! :)

Can we add a test case to verify this works on the TestProject?